ext/curl/sync-constants.php

```diff
@@ -10,17 +10,87 @@
 
 const CURL_DOC_FILE = 'https://curl.haxx.se/libcurl/c/symbols-in-versions.html';
 
-const SOURCE_FILE = __DIR__ . '/interface.c';
+const SOURCE_FILE = __DIR__ . '/curl_arginfo.h';
 
 const MIN_SUPPORTED_CURL_VERSION = '7.29.0';
 
-const IGNORED_CONSTANTS = [
+const IGNORED_CURL_CONSTANTS = [
     'CURLOPT_PROGRESSDATA',
-    'CURLOPT_XFERINFODATA'
+    'CURLOPT_XFERINFODATA',
+];
+
+const IGNORED_PHP_CONSTANTS = [
+    'CURLOPT_BINARYTRANSFER',
+    'CURLOPT_RETURNTRANSFER',
+    'CURLOPT_SAFE_UPLOAD',
 ];
 
 const CONSTANTS_REGEX_PATTERN = '~^CURL(?:OPT|_VERSION)_[A-Z0-9_]+$~';
 
+/**
+ * A simple helper to create ASCII tables.
+ * It assumes that the same number of columns is always given to add().
+ */
+class AsciiTable
```

Comment:
had to move the declaration to the top, otherwise the class was not found on my system